matrix-arithmetic.Rmd

```diff
@@ -28,21 +20,12 @@ $$
 \overline{B} \ \ {\small +}{=} \ \ \overline{C}
 $$
 
-
 ## Subtraction
 
 $$
 C = A - B
 $$
 
-### Derivatives {-}
```

Comment:
Why do you want to remove the simple derivatives?

Reply:
Just for consistency, I thought it might seem strange to have them only for some cases. I'll put them back in if you prefer.